assets/src/edit-story/components/panels/design/filter/karma/filter.karma.js

```diff
@@ -108,32 +108,32 @@ describe('Filter Panel', () => {
     });
 
     it('should render correct overlay when clicking "solid"', async () => {
-      await fixture.events.click(filterPanel.solid.button);
+      await fixture.events.click(filterPanel.solid);
 
       const overlay = await waitFor(getBackgroundElementOverlay);
       expect(overlay).toBeTruthy();
-      expect(overlay).toHaveStyle('background-color', 'rgba(0, 0, 0, 0.3)');
+      expect(overlay).toHaveStyle('background-color', 'rgba(0, 0, 0, 0.5)');
     });
 
     it('should render correct overlay when clicking "linear"', async () => {
-      await fixture.events.click(filterPanel.linear.button);
+      await fixture.events.click(filterPanel.linear);
 
       const overlay = await waitFor(getBackgroundElementOverlay);
       expect(overlay).toBeTruthy();
       expect(overlay).toHaveStyle(
         'background-image',
-        'linear-gradient(rgba(0, 0, 0, 0) 40%, rgba(0, 0, 0, 0.9) 100%)'
+        'linear-gradient(rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.63) 100%)'
```

Comment:
New defaults?

Reply:
Yep, as far as I could see but I'll double-check the values.

Reply:
Ok, looks like I had incorrect values here as a result due to alpha and the `size` -- updated these too, should match Figma now.